viscy/translation/engine.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self.l2_alpha = l2_alpha
         self.ms_dssim_alpha = ms_dssim_alpha
 
-    @torch.amp.custom_fwd(device_type="cuda", cast_inputs=torch.float32)
+    @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)
```

Comment:
Merge error.

Reply:
I'm having a hard time understanding this. #151 is merged and released with v0.2.1. Why are you changing this in this PR?

Reply:
Test won't pass if I don't change this. I can revert the change but then you'd have to be aware that some of the tests won't pass on this PR.

Reply:
This is because your branch is not up to date with the base branch. `train-linear-classifier` already have this:
https://github.com/mehta-lab/VisCy/blob/4fe26108cf14bc17a4b3c8b3c484eb6ef26f1307/viscy/translation/engine.py#L71